README.md

```diff
@@ -11,7 +11,9 @@ You can install Pop CLI as follows:
 ```shell
 cargo install --locked --git https://github.com/r0gue-io/pop-cli
 ```
-> For detailed instructions on how to install Pop CLI, please refer to our documentation: https://learn.onpop.io/pop/v/pop-cli/welcome/installing-pop-cli
+
+> For detailed instructions on how to install Pop CLI, please refer to our
+> documentation: https://learn.onpop.io/pop/v/pop-cli/welcome/installing-pop-cli
```

Comment:
Note that I fixed this in https://github.com/r0gue-io/pop-cli/pull/159

Reply:
Merged